### ISSUE TITLE
Stock Kotlin Container classes should extend ViewGroup, not FrameLayout

### ISSUE THREAD

**Issue by rjrjr** [square/workflow#670]
Extending FrameLayout is just sloppy, implies general use that is not supported.

**Comment by wugeorgeq** [comment]
Looking at `BackStackContainer` in `workflow-ui:backstack-android` and `WorkflowLayout` in `core-android`. Seems like these are intended to display one child view at a time, would it make sense to override `onLayout` like so (it's an abstract function in ViewGroup)? 
```
override fun onLayout(
    changed: Boolean,
    left: Int,
    top: Int,
    right: Int,
    bottom: Int
  ) {
    currentView?.layout(0, 0, 0, 0)
  }
```

**Comment by rjrjr** [comment]
Yes, exactly this. And IIRC you need to implement `onMeasure` too.

I think that's true of every `FrameLayout` subclass in the library, I was just lazy.

**Comment by rjrjr** [comment]
See `ScrimContainer` for an example.

**Comment by zach-klippenstein** [comment]
Should we make a `PassthroughLayout` class?

**Comment by rjrjr** [comment]
My gut is that it would be overkill. The copy / paste is so minimal.

**Comment by rjrjr** [comment]
Fixed in #1193
